client/src/modules/prices/prices.js

```diff
@@ -135,11 +134,17 @@ function PriceListController(
       });
   }
 
-  function download() {
+  function download(renderer) {
+    const displayNames = columnConfig.getDisplayNames();
     const options = {
-      renderer : 'pdf',
+      renderer,
       lang : Languages.key,
+      detailed : 1,
     };
+
+    if (renderer === 'xlsx') {
```

Comment:
We should probably rename keys if option is not `pdf`.  CSV will want renamed keys as well.

Reply:
okay